tokens/custom-spectrum/custom-dark-vars.css

```diff
@@ -12,11 +12,6 @@ governing permissions and limitations under the License.
 /* This file contains overrides and additions to core tokens */
 
 .spectrum--dark {
-  --spectrum-menu-item-background-color-default: rgba(255, 255, 255, 0); /* --spectrum-gray-900 */
-  --spectrum-menu-item-background-color-hover: var(--spectrum-transparent-white-200);
-  --spectrum-menu-item-background-color-down: var(--spectrum-transparent-white-200);
-  --spectrum-menu-item-background-color-key-focus: var(--spectrum-transparent-white-200);
```

Comment:
Did these just move into the menu's `index.css` instead then?

Reply:
These are coming from the tokens now! See https://github.com/adobe/spectrum-css/pull/2802#discussion_r1633830351